frontend/public/components/modals/index.ts

```diff
@@ -54,3 +59,9 @@ export const clusterChannelModal = (props) => import('./cluster-channel-modal' /
 
 export const clusterUpdateModal = (props) => import('./cluster-update-modal' /* webpackChunkName: "cluster-update-modal" */)
   .then(m => m.clusterUpdateModal(props));
+
+export const taintUpdateModal = (props) => import('./taint-update-modal' /* webpackChunkName: "taint-update-modal" */)
+  .then(m => m.taintUpdateModal(props));
```

Comment:
nit: I'd probably call these `taintsModal` and `tolerationsModal` (like `labelsModal` above). `clusterUpdateModal` has "update" in the name because you're upgrading the cluster